style.css

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Styles in IE and Edge don't work as intended.
![MicrosoftTeams-image (12)](https://user-images.githubusercontent.com/53054365/94273974-c9b70180-ff4d-11ea-8cfa-878379c2eea0.png)

Reply:
fixed with new commit

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/53054365/95216347-9fd9c680-07fa-11eb-9316-d00a136969ce.jpg)

Reply:
Please fix the width of the part highlighted with red in the screenshot.

Reply:
Thanks for the tip! The issue is now fixed.